ui_framework/components/tabs/tabs.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTab } from './tab';
+
+export const KuiTabs = ({
+  children,
+  selectedTabIndex,
+  onSelectedTabChanged,
+  className,
+  ...rest
+}) => {
+  const classes = classNames('kuiTabs', className);
+  const tabs = children.map((child,index)=>(
```

Comment:
@PopradiArpad Could you check that you have the latest npm dependencies installed? Deleting `node_modules` and running `npm install` will ensure everything is up to date too. The linter has been falling out of date for some people recently.